code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -65,3 +67,41 @@
 
 	toggle_scope(2.0)
 
+////////////// Dragunov Sniper Rifle //////////////
+
+/* // Commented out until it's not worthless.
+/obj/item/weapon/gun/projectile/SVD
+	name = "\improper Dragunov"
+	desc = "The SVD, also known as the Dragunov, was mass produced with an Optical Sniper Sight so simple that even Ivan can figure out how it works. Too bad for you that it's written in Russian. Uses 7.62mm rounds."
+	icon_state = "SVD"
+	item_state = "SVD"
+	w_class = 5 // So it can't fit in a backpack.
+	force = 10
+	slot_flags = SLOT_BACK // Needs a sprite.
+	origin_tech = list(TECH_COMBAT = 8, TECH_MATERIAL = 2, TECH_ILLEGAL = 8)
+	recoil = 2 //extra kickback
+	caliber = "a762"
+	load_method = MAGAZINE
+	accuracy = -3 //shooting at the hip
+	scoped_accuracy = 0
+//	requires_two_hands = 1
+	one_handed_penalty = 4 // The weapon itself is heavy, and the long barrel makes it hard to hold steady with just one hand.
+	fire_sound = 'sound/weapons/SVD_shot.ogg'
+	magazine_type = /obj/item/ammo_magazine/SVD
+	allowed_magazines = list(/obj/item/ammo_magazine/SVD, /obj/item/ammo_magazine/c762)
+
+/obj/item/weapon/gun/projectile/SVD/update_icon()
+	..()
+	/*if(istype(ammo_magazine,/obj/item/ammo_magazine/c762)
+		icon_state = "SVD-bigmag"*/ //No icon for this exists yet.
+	if(ammo_magazine)
+		icon_state = "SVD"
+	else
+		icon_state = "SVD-empty"
+
+/obj/item/weapon/gun/projectile/SVD/verb/scope()
+	set category = "Object"
+	set name = "Use Scope"
+	set popup_menu = 1
+
+	toggle_scope(2.0)*/
```

Comment:
I think you could/should stretch the commenting down to here, and comment out the ammo for it, which is still in but has nothing to do.